cached-webpgr.js

```diff
@@ -19,95 +19,113 @@
  * It will make an ajax call to retrive the desired script from the provided url and store it
  * in the localStorage under the provided name. The stored script will be wrapped like in this example:
  * `{content: '// scrip content $(document).ready(...)', version: '1.02.03'}`
- * @param {string} name localStorage identifier; shoud be the same as on the server-side
- * @param {string} url `path/to/script.js`; shoud be on the same server (or with remove with CORS header access-control-allow-origin:*)
+ * @param {string} url (see `requireScript` or 'css')
+ * @param {string} name (see `requireScript` or 'css')
+ * @param {string} version (see `requireScript` or 'css')
+ * @param {Boolean} css (false =js, true=css)
```

Comment:
I think a string would be better here so the API is clearer so I would change this to `type: ['css' || 'js]`, or we parse the url suffix, which should be even better since the API would not change.
```
var extension = url.split('.').pop()
if (extension === 'js') ...
if (extension === 'css') ...
```

Reply:
* German
habe mal "cached-webpgr" zerlegt und zu einer version 2 überarbeitet!
js und css werden unterstütz!
vielleicht ist die Ideen/Überarbeitung Ansatz interessant für dich.
* English translation by google
I disassembled "cached webpgr" and belonged to version 2!
js and css are processed!
Perhaps the ideas / revision approach is interesting for you.


[cached-webpgr-alpha-v2](https://github.com/WarManiac/cached-webpgr-v2)